resources/testharness.js

```diff
@@ -1723,7 +1816,7 @@ policies and contribution forms [3].
         this.message_target = message_target;
         this.message_handler = function(message) {
             var passesFilter = !message_filter || message_filter(message);
-            if (this_obj.running && message.data && passesFilter &&
+            if (message.data && passesFilter &&
```

Comment:
What's this change for? I notice I'm not getting most of the changes in RemoteTest, can you describe in the commit message how it's impacted?

Reply:
This is superfluous because the property is initialized to `true` and only set
to `false` immediately after the callback function is removed.

That said, I really shouldn't be making unrelated simplifications in this
patch. I'll replace the removal with a "TODO" that suggests the change.

Reply:
Thanks!